apps/meteor/tests/e2e/omnichannel/omnichannel-agents.spec.ts

```diff
@@ -1,44 +1,104 @@
+	});
+
+	test('OC - Manage Agents - Add, search and remove using table', async ({ page }) => {
+		await test.step('expect "user1" be first ', async () => {
+			await poOmnichannelAgents.inputUsername.type('user');
+			await expect(page.locator('role=option[name="user1"]')).toContainText('user1');
+
+			await poOmnichannelAgents.inputUsername.type('');
+		});
+
+		await test.step('expect add "user1" as agent', async () => {
+			await poOmnichannelAgents.inputUsername.type('user1');
+			await page.locator('role=option[name="user1"]').click();
+			await poOmnichannelAgents.btnAdd.click();
+
+			await poOmnichannelAgents.inputSearch.fill('user1');
+			await expect(poOmnichannelAgents.firstRowInTable).toBeVisible();
+		});
+
+		await test.step('expect remove "user1" as agent', async () => {
+			await poOmnichannelAgents.inputSearch.fill('user1');
+			await poOmnichannelAgents.btnDeletefirstRowInTable.click();
+			await poOmnichannelAgents.btnModalRemove.click();
+
+			await poOmnichannelAgents.inputSearch.fill('user1');
+			await expect(poOmnichannelAgents.firstRowInTable).toBeHidden();
+		});
+	});
+
+	test('OC - Manage Agents - Edit and Remove', async ({ page }) => {
 		await poOmnichannelAgents.inputUsername.type('user1');
 		await page.locator('role=option[name="user1"]').click();
 		await poOmnichannelAgents.btnAdd.click();
 
-		await poOmnichannelAgents.inputSearch.fill('user1');
-		await expect(poOmnichannelAgents.firstRowInTable).toBeVisible();
-	});
+		await test.step('expect update "user1" information', async () => {
+			await poOmnichannelAgents.inputSearch.fill('user1');
+			await poOmnichannelAgents.firstRowInTable.click();
+			await poOmnichannelAgents.btnEdit.click();
 
-	test('expect update "user1" status', async ({ page }) => {
-		await poOmnichannelAgents.inputSearch.fill('user1');
-		await poOmnichannelAgents.firstRowInTable.click();
+			await poOmnichannelAgents.StatusSelect.click();
+			await page.locator(`.rcx-option__content:has-text("Not available")`).click();
+			await poOmnichannelAgents.DepartmentSelect.click();
+			await page.locator(`.rcx-option__content:has-text("${department.name}")`).click();
+			await poOmnichannelAgents.btnSave.click();
+		});
 
-		await poOmnichannelAgents.btnEdit.click();
-		await poOmnichannelAgents.btnStatus.click();
-		await page.locator(`.rcx-option__content:has-text("Not available")`).click();
-		await poOmnichannelAgents.btnSave.click();
+		await test.step('expect removing "user1" via sidebar', async () => {
+			await poOmnichannelAgents.inputSearch.fill('user1');
+			await poOmnichannelAgents.firstRowInTable.click();
+			await poOmnichannelAgents.btnRemove.click();
+		});
 	});
 
-	test('expect remove "user1" as agent', async () => {
-		await poOmnichannelAgents.inputSearch.fill('user1');
-		await poOmnichannelAgents.btnDeletefirstRowInTable.click();
-		await poOmnichannelAgents.btnModalRemove.click();
+	test('OC - [EE] Manage Agents - Edit ', async ({ page }) => {
```

Comment:
This test seem to be incomplete 🤔 
What are we testing here? I don't see a counterpart in the CE test.